examples/nextjs-v14-app/README.md

```diff
@@ -0,0 +1,9 @@
+# Lit in Next.js example
```

Comment:
When testing this locally with `JavaScript Disabled`, the "Hello, Friend!" component renders, but the component beneath doesn't seem to render.
My expectation is that two components would SSR with DSD.


Edit: Could this be that I need to link to the new packages?

Reply:
It appears to show fine for me. Depending on how you ran the example.. if you're doing this outside the monorepo, you'd need `@lit-labs/ssr-react`, and `@lit/react` also linked to the local copy, as well as `@lit-labs/nextjs`.

But run it within this branch of the monorepo and everything should already link to the monorepo copy.